### ISSUE TITLE
Project not compiling lessOptions

### ISSUE THREAD

**Issue by henryw14** [gdub22/ember-cli-less#19]
Just added someone new to our dev team. She installed Yosemite, while I have Mavericks. We are using the ember-cli-less to compile our less, using the `lessOptions` hash inside of our brocfile to specify the order of compilation. Anyway, when my app builds, I see this output:
```
Slowest Trees                  | Total
-------------------------------+----------------
Concat: Vendor                 | 2829ms
Babel                          | 1167ms
ES3SafeFilter                  | 925ms
LessCompiler                   | 885ms
JSHint app- QUnit              | 632ms
ES6: App Tree                  | 498ms
```
whereas when she runs it, she sees:
```
Slowest Trees                                 | Total
----------------------------------------------+---------------------
Concat: Vendor                                | 2744ms
Babel                                         | 1337ms
ES3SafeFilter                                 | 687ms
JSHint app- QUnit                             | 616ms

Slowest Trees (cumulative)                    | Total (avg)
----------------------------------------------+---------------------
Concat: Vendor (1)                            | 2744ms
Babel (2)                                     | 1358ms (679 ms)
ES3SafeFilter (1)                             | 687ms
JSHint app- QUnit (1)                         | 616ms
```
which is clearly missing the less compilation step and explains why she doesn't have the app_name.css file in here `dist/assets` folder. Not sure that the OS.X versions are the culprit, but I do not know what else could be the issue? Both of us are running Ember Cli `version: 0.2.0-beta.1`

**Comment by gdub22** [comment]
@henryw14 that output is the 'slowest' trees, not necessarily all the trees.  Not sure what else could be wrong without seeing more of your setup.  OS X version shouldn't matter.  I'd recommend updating ember-cli out of the beta version as a first step.

**Comment by henryw14** [comment]
Turns out she was using 1.3.0 of Ember-cli-less, and we were using 1.2.0. Upgrading her version of ember-cli did not fix it, but downgrading back ember-cli-less to 1.2.0 did. I don't really have much to help you reproduce this other than what I told you above so you might as well close it

**Comment by gdub22** [comment]
@henryw14 I just released 1.3.1 which should work well for both of you.  1.3.0 had a regression where the lessOptions were not being read.  Sorry for the confusion.
